WordPress/src/main/java/org/wordpress/android/ui/domains/DomainRegistrationDetailsFragment.kt

```diff
@@ -293,15 +293,23 @@ class DomainRegistrationDetailsFragment : Fragment() {
                 country_input, address_first_line_input, city_input, postal_code_input
         )
 
+        var fieldToFocusOn: TextInputEditText? = null
```

Comment:
Just note: Ideally this business logic would be encapsulated in the VM so it can be tested. However, let's not refactor it in this PR as the number of changes would be significant.